usecases.md

```diff
@@ -3,3 +3,434 @@ layout: base
+
+**Requirements** :
+
+- All requirements as listed for Use Case 1.A or 1.B for each forecast in the comparison.
+- User can share forecast specifications (e.g., quantity, interval, horizon, averaging, lead times) with other framework users and invite other framework users to share forecasts for the comparison.
+- Framework supports anonymous forecast comparison.
+- Framework protects forecasts as specified by each forecast owner.
+- Framework provides metric values for the comparison among forecasts, a visual display of comparison results, and a report for download.
+
+### 1.D. Compare forecasts to measurements for sites and aggregates {#uc1D}
+
+**Use case narrative** : A framework administrator defines an aggregate of locations. A framework user uploads forecasts for locations in the aggregate and a forecast for the aggregation of locations. A framework user uploads corresponding data or selects reference data, specifies filters, specifies temporal averaging, and selects metrics. The forecasts and data may represent individual point sensors, power plants, or aggregated quantities. The framework calculates metrics and provides a visual display comparing the forecasts across scales.
+
+**Requirements** :
+
+- All requirements as listed for Use Case 1.A, 1.B, or 1.C, as appropriate.
+- User uploads an aggregation of forecasts with weights specified by the framework.
+- Framework computes observations, as specified.
+- Framework provides metric values for the comparison among forecasts, a visual display of comparison results, and a report for download.
+
+### 1.E. Evaluate an event forecast {#uc1E}
+
+**Use case narrative** : A framework user can analyze an event forecast.
+
+**Requirements** :
+
+- All requirements as listed by Use Case 1.A, 1.B, or 1.C, as appropriate.
+- User can upload an event forecast, e.g., a time series of true/false values, or a time series of event occurrence likelihood.
+- User can upload corresponding event data or connect to a saved event time series (2.B).
+- User can select among event forecast performance metrics (3.A).
+- Framework provides metric values and visuals of event forecast performance (3.A).
+- Framework provides a report for user download.
+
+### 1.F. Conduct forecast trial {#uc1F}
+
+**Use case narrative** : The framework administrator, in consultation with trial participants, defines trial period, locations, forecast quantities, time horizons, time resolutions, etc. of a forecast trial. The trial may be either retrospective or live, and may involve a test period followed by an evaluation period.
+
+**Requirements** :
+
+- All requirements as listed for Use Case 1.C
+- Administrator, in consultation with trial participants, defines data sources, forecast run intervals/horizons, trial period, metrics, filters
+- Framework concatenates regularly posted forecast runs into a continuous forecast.
```

Comment:
Does this mean you can only have one forecast for each valid time, given the data is concatenated? Seems like you would need 1G to do something where you had forecaster provide forecasts for the same valid time at different lead times? If 1G doesn’t happen, people would have to do multiple trials (e.g. one for DA, one for HA, etc.), and then do their own combining afterward?

Reply:
Yes, Aidan highlighted concatenates here. I agree that we are not clear about we'd handle "trials" of different parameters within a "multi-trial". I had been thinking of them as entirely separate trials. I don't know if it's worth the effort to group together trials into a "multi-trial" (awful terminology but I hope it gets the point across).

Reply:
added the following to the use case narrative: "Separate trials may be administered for multiple forecast parameters (e.g. hour ahead and day ahead forecast evaluations for the same set of locations)."